lms/djangoapps/static_template_view/tests/test_views.py

```diff
@@ -24,3 +25,35 @@ def test_about(self):
         Test the about view
         """
         self._test_view('about', 'text/html')
+
+    def test_404(self):
+        """
+        Test the 404 view
+        """
+        url = reverse('static_template_view.views.render_404')
+        resp = self.client.get(url)
+        self.assertEqual(resp.status_code, 200)
+        self.assertEqual(resp['Content-Type'], 'text/html')
+
+        # check response with branding
+        resp = self.client.get(url, HTTP_HOST=settings.MICROSITE_TEST_HOSTNAME)
+        self.assertContains(resp, settings.MICROSITE_CONFIGURATION['test_microsite']['email_from_address'])
+
+    def test_500(self):
+        """
+        Test the 500 view
+        """
+        url = reverse('static_template_view.views.render_500')
+        resp = self.client.get(url)
+        self.assertEqual(resp.status_code, 500)
+        self.assertEqual(resp['Content-Type'], 'text/html; charset=utf-8')
+
+        # check response with branding
+        resp = self.client.get(url, HTTP_HOST=settings.MICROSITE_TEST_HOSTNAME)
+        self.assertContains(
```

Comment:
We need to also test for the default condition that returns the tech support email address